docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
-# Dynamic Multi-Criteria Decision Analysis
\ No newline at end of file
+# Dynamic Multi-Criteria Decision Analysis
+
+## Site Selection in ADRIA
+At each time step where an intervention is performed at a site, the sites for intervention can be selected either randomly or by using a site selection algorithm. When an algorithm is used, site selection is performed by iteratively updating how each site/reef in the domain performs against a range of criteria. Currently, ADRIA allows the following criteria to be used in site selection:
+
+* Connectivity, calculated as ```centrality*(site coral cover)*(site area)```.
+* Wave damage probability.
+* Heat stress probability.
+* Priority predecessors, or sites with the strongest connectivity to "priority sites" (defined by the user in the ```prioritysites``` parameter).
+* Priority zones, such as GBRMPA management zones (defined by the user in the ```priorityzones``` parameter in order of preference).
+* Coral cover relative to the site carrying capacity.
+* Site depth, for which median site depth is used.
+
+In the case of site selection for seeding, the coral cover criterium is defined as the area not occupied by coral relative to the site carrying capacity. For shading, the coral cover criterium is defined as the area already covered with coral relative to the carrying capacity. All other criteria are identical for seeding and shading, so that sites selected for these two interventions tend to be similar. The priority predecessors and priority zones criteria allow the user to incorporate spatial objectives, such as through GBRMPA management zone layers or data layers from other spatial management software such as MARXAN.
+
+These criteria are used within methods based on multi-criteria decision analysis (MCDA) to rank sites from most to least suitable for intervention at each intervention time step when ADRIA is run. The criteria are weighted according to the user's preference and the weights can be defined in ADRIA through the input scenario parameters, or inline, as in the following example:
+
+```julia
+@info "Loading data package"
+here = @__DIR__
```

Comment:
Example code snippets should be succinct as possible, and either:

1. Work if the reader runs the copy/pasted code, or
2. Indicate that it is for example purposes only (implicitly is okay)

This is lifted from the example scripts which expects the file to be run from the `examples` directory, which is why the path to the domain data package is relative to `here` (intended to be the example directory).

For "normal" users, they will have installed ADRIA from the package manager and may not even know where it gets installed to, so it (1) won't work, and (2) looks more complicated than it actually is.

I would simply use:

```julia
dom = ADRIA.load_domain("path to a domain data package")
```

or use comments:

```julia
# Replace `PATH_TO_DATA` with a string pointing to the domain data package
dom = ADRIA.load_domain(PATH_TO_DATA)
```

Also, remove the `@info` statements and replace with comments if they are needed.